lib/pbench/server/api/resources/query_apis/datasets/datasets_mappings.py

```diff
@@ -118,6 +115,6 @@ def _get(self, json_data: JSON, request: Request) -> Response:
             return jsonify(result)
         except TemplateNotFound:
             self.logger.exception(
-                "Document template {} not found in the database.", index_name
+                "Document template {} not found in the database.", index["index"]
```

Comment:
Since we are making a change here, can we perhaps update the exception string to add clarity?  Perhaps something like, "Document template for index {} not found in the database"?